client/web/src/keyboardShortcuts/KeyboardShortcutsHelp.test.tsx

```diff
@@ -1,29 +1,38 @@
+import { ShortcutProvider } from '@slimsag/react-shortcuts'
+import { render, fireEvent, waitFor, screen } from '@testing-library/react'
 import React from 'react'
-import renderer from 'react-test-renderer'
-import { Modal } from 'reactstrap'
 
 import { KeyboardShortcutsHelp } from './KeyboardShortcutsHelp'
 
 describe('KeyboardShortcutsHelp', () => {
-    test('', () => {
-        const output = renderer.create(
-            <KeyboardShortcutsHelp
-                keyboardShortcuts={[
-                    {
+    test('', async () => {
```

Comment:
```suggestion
    test('is triggered correctly', async () => {
```

nit: having an empty test name feels wrong, so let's update this too :)